### DISCUSSION TITLE
how to get hls url?

### DISCUSSION THREAD

**Discussion by Jamong435** [OvenMediaLabs/OvenMediaEngine#1112] [Help]
how to get hls url?

obs input value :

server : rtmp://192.168.40.110:1935/app
stream key : stream

webrtc test url : 
ws://192.168.40.110:3333/app/stream [ok]
ws://192.168.40.110:3333/app/stream?transport=tcp [ok]

http://192.168.40.110:8080/app/stream/playlist.m3u8 [no]
https:/192.168.40.110:443/app/stream_o/playlist.m3u8 [no]

thank you :+1:

**Comment by getroot** [comment]
https://airensoft.gitbook.io/ovenmediaengine/streaming/low-latency-hls#streaming

**Comment by Jamong435** [comment]
Does it have to have a domain to work?

**Comment by umeshshende** [comment]
@Jamong435  Try this,  In my case i have tried http url without hostname it work , I have played on VLC and RTMP publish from OBS

http://192.168.40.110:8080/app/stream/llhls.m3u8 
https:/192.168.40.110:443/app/stream_o/llhls.m3u8

**Comment by getroot** [comment]
Depending on the security policy of the browser you are using, playback may not be possible if there is no certificate. Try playing with your IP and non-tls address, if that doesn't work, check your web browser's console.

Players like VLC will play just fine because they don't have such a security policy. However, since VLC does not support LLHLS yet, it will play in legacy HLS (long latency).

**Comment by Jamong435** [comment]
It's running on centos7.

If I run ovenmediaengine , the 8080 port number won't open.

Proto Recv-Q Send-Q Local Address           Foreign Address         State
tcp        0      0 0.0.0.0:3478            0.0.0.0:*               LISTEN
tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN
tcp        0      0 0.0.0.0:3333            0.0.0.0:*               LISTEN
tcp        0      0 0.0.0.0:3334            0.0.0.0:*               LISTEN
tcp        0      0 127.0.0.1:9000          0.0.0.0:*               LISTEN
tcp        0      0 127.0.0.1:42187         0.0.0.0:*               LISTEN
tcp        0      0 0.0.0.0:1935            0.0.0.0:*               LISTEN
tcp        0      0 0.0.0.0:111             0.0.0.0:*               LISTEN
tcp6       0      0 :::80                   :::*                    LISTEN
tcp6       0      0 :::21                   :::*                    LISTEN
tcp6       0      0 :::22                   :::*                    LISTEN
tcp6       0      0 :::33060                :::*                    LISTEN
tcp6       0      0 :::3306                 :::*                    LISTEN
tcp6       0      0 :::111                  :::*                    LISTEN



Is it okay without the 8080 port number?

**Comment by getroot** [comment]
The LLHLS port depends on your setup.

https://airensoft.gitbook.io/ovenmediaengine/streaming/low-latency-hls#configuration

**Comment by Jamong435** [comment]
I'm sorry, but I don't know which part to change. Is there an explanation about the file? or Can you tell me the location of the file?

**Comment by getroot** [comment]
The default port is 3333/3334 unless you have modified Server.xml.
You can play with http://ip:3333/app/stream/llhls.m3u8. However, modern browsers may not accept external links without a certificate.
If you want to play with https://domain:3334/app/stream/llhls.m3u8, you must modify Server.xml to install a certificate and connect the domain.

**Comment by getroot** [comment]
https://airensoft.gitbook.io/ovenmediaengine/configuration

**Comment by gregfr** [comment]
Why do you try to access ports 443 and 8080? The documentation says:
```
docker run -d \
-p 1935:1935 -p 4000:4000/udp -p 3333:3333 -p 3334:3334 -p 3478:3478 -p 9000:9000 -p 9999:9999/udp \
airensoft/ovenmediaengine:0.15.0
```

so it's normal that port 8080 and 443 are not open.

Is your purpose to stream on your local network or be accessible worldwide?
